app/workers/tasks.py

```diff
@@ -55,6 +55,7 @@ async def get_contract_metadata_task(
             address,
             chain_id,
         )
+        # TODO Check if contract is MultiSend. In that case, get the transaction and decode it
```

Comment:
The check should be here or in the rabbitmq consumer? 
Other topic is that we are not checking if a contract is really a contract, we are trying to download every address. Should we check this first?

Reply:
I'm not saying that we should do it here, but we should be aware: https://github.com/safe-global/safe-decoder-service/issues/67